dftimewolf/lib/collectors/gcloud.py

```diff
@@ -161,6 +162,9 @@ def SetUp(self,
             value=analysis_vm_name))
 
     try:
+      if self.remote_instance_name:
+        self.remote_project.compute.GetInstance(self.remote_instance_name)
+
```

Comment:
it's always better to make the try / except block as small as possible around the code that can actually raise the exception. in this case, binging up the block in L183 to L167:

```python
if self.remote_instance_name:
  try:
    self.remote_project.compute.GetInstance()
  except ResourceNotFoundError as exception:
    #...
```

Also, I would move this whole block to be one of the first functions to be executed in SetUp (so we don't create the analysis VM for nothing if we don't have access to the target instance or if it doesn't exist).

Finally, I'd extract this to its own function within the module (and to be frank this whole SetUp function could use some refactoring - for a future PR maybe)